OpenDreamClient/Interface/Controls/ScalingViewport.cs

```diff
@@ -228,7 +228,7 @@ private void RegenerateViewport()
                 ViewportSize * renderScale,
                 new TextureSampleParameters
                 {
-                    Filter = StretchMode == ScalingViewportStretchMode.Bilinear,
+                    Filter = StretchMode == ScalingViewportStretchMode.Nearest,
```

Comment:
Bilinear filtering should only be used when using bilinear stretching
```suggestion
                    Filter = StretchMode == ScalingViewportStretchMode.Bilinear,
```